docs/contributing.rst

```diff
@@ -61,6 +61,34 @@ Outside of data compatibility, follow the usual semver rules for API changes and
 to a major release.
 
 
+The Scope of the Library
+------------------------
+
+The purpose of this library is to provide a Pythonic ORM layer on top of DynamoDB to be used
```

Comment:
ODM not ORM

Reply:
agree, we just say "ORM-like" in the README so I thought...

Reply:
can we just change the readme as well -- we don't support any relationship management in this library

Reply:
or just drop the bullet point from the features list entirely